### PR TITLE
Feature/351

### PR DESCRIPTION
**Pull request by eboyer**
#351 

*I'd like to write a little documentation around this but wanted to open up the review!

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 95d35345968f24895f92be243516160f5f3df6b9

https://deploy-preview-492--fundamental.netlify.com

File: scss/components/menu.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+$block: #{$fd-namespace}-menu;
+
+.#{$block} {
+  //SETTINGS
+    //LOCAL VARS (set all themeable properties, always include !default)
+    $fd-menu-color: solid 1px $fd-forms-color !default;
+    $fd-menu-border: solid 1px $fd-forms-border-color !default;
+    $fd-menu-z-index: 2 !default;
```
Comment (saadmhybris) [inline scss/components/menu.scss]: local var defined but, not used anywhere.

File: scss/components/menu.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    $fd-menu-background-color: fd-color(background) !default;
+    $fd-menu-shadow: 0 0 4px 1px rgba(fd-color(neutral, 2), 1) !default;
```
Comment (saadmhybris) [inline scss/components/menu.scss]: local var defined but, not used anywhere.

File: scss/components/menu.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    display: inline-block;
+    min-width: 100%;
+    white-space: nowrap;
+    background-color: $fd-menu-background-color;
+    padding-top: fd-space(base);
+    padding-bottom: fd-space(base);
+
+    &__group,
+    &__list {
+      //background-color: $fd-menu-background-color;
```
Comment (saadmhybris) [inline scss/components/menu.scss]: remove commented code

File: scss/components/popover.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
+@import "./../settings";
+@import "./../mixins";
+@import "./../icons/mixins";
+@import "./../functions";
+@import "./mixins";
+
+$block: #{$fd-namespace}-popover;
+.#{$block} {
+    //LOCAL VARS (set all themeable properties, always include !default)
+    $fd-popover-color: solid 1px $fd-forms-color !default;
```
Comment (saadmhybris) [inline scss/components/popover.scss]: local var defined, but not used

File: test/resources/controls.js [hunk 5]
```diff
@@ -14,7 +14,7 @@
                 //remove or hide if we want some animation
                 // target.setAttribute("aria-hidden", true);
                 target.remove();
-                return;
+                return;``
```
Comment (saadmhybris) [inline test/resources/controls.js]: empty quotes ``

**Pull request merged**